django-reference-app/django-ref-config/settings.py

```diff
@@ -20,25 +20,29 @@
 # See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'p@rjsk4a8kwv0t@b5@gvf$(i*8)=-5zn=*r*#1n01njuxzw+=y'
+SECRET_KEY = 'd^2sg(rd9&3s$uj46-di0i_14bcy84x$lxsjt)d0ur_h!q*_!w'
 
 # SECURITY WARNING: don't run with debug turned on in production!
```

Comment:
Important note here - when `DEBUG = False`, `manage.py runserver` does not serve static assets, hence the need for a proxy webserver in front of the Django app.